lib/xlibc-rt/_alldiv.s

```diff
@@ -0,0 +1,158 @@
+// This file is dual licensed under the MIT and the University of Illinois Open
+// Source Licenses. See LICENSE.TXT for details.
+
+
+// di_int __divdi3(di_int a, di_int b);
+
+// result = a / b.
+// both inputs and the output are 64-bit signed integers.
+// This will do whatever the underlying hardware is set to do on division by zero.
+// No other exceptions are generated, as the divide cannot overflow.
+//
+// This is targeted at 32-bit x86 *only*, as this can be done directly in hardware
+// on x86_64.  The performance goal is ~40 cycles per divide, which is faster than
+// currently possible via simulation of integer divides on the x87 unit.
+//
+// Stephen Canon, December 2008
+// Stefan Schmidt, October 2018
```

Comment:
I didn't write the code, I only modified it to fit the required calling convention (cdecl->stdcall) and adjusted the name (divdi3->_alldiv).
My intention was to make sure that people see that I've modified the code, so the original author doesn't get blamed in case somethings broken.
I see it's misleading and I'll remove it.